osidb/signals.py

```diff
@@ -117,3 +117,14 @@ def update_local_updated_dt_tracker(sender, instance, **kwargs):
             no_alerts=True,  # recreating alerts from nested entities can cause deadlocks
             raise_validation_error=False,
         )
+
+
+@receiver(pre_save, sender=Affect)
+def update_last_impact_increase_dt(sender, instance, **kwargs):
+    if not instance._state.adding and Impact(instance.impact) > Impact(
+        Affect.objects.get(pk=instance.pk).impact
+    ):
+        for tracker in instance.trackers.all():
+            if Impact(instance.impact) > tracker.aggregated_impact:
+                tracker.last_impact_increase_dt = timezone.now()
+                tracker.save(auto_timestamps=False, raise_validation_error=False)
```

Comment:
As we do not need to auto-set timestamps or run validations I am thinking whether this would not be a case when **update** can be easily used instead of **save**. The reasons are described in https://docs.google.com/document/d/1-3bCyQky6c0OboSHRVEFm4BSl3nM79LHIVXfQ8HyRqc/edit?usp=sharing and I do not remember then in detail :smile: but **save** can have some unintended side effects together with the parallelism.

Reply:
Makes sense, I changed it